### DISCUSSION TITLE
SyntaxError: Unexpected token < in JSON at position 0

### DISCUSSION THREAD

**Discussion by mkraibt** [spatie/pdf-to-text#51] [Q&A]
Hi, I have been experimenting with this package, and works like a charm when it does but I constantly get `SyntaxError: Unexpected token < in JSON at position 0` error, though sometimes I do get the content of PDF; other times, I have been getting empty strings. Anything I am missing here. Again thanks for this plugin.

**Comment by freekmurze** [comment]
For most PDFs it should just work. Feel free to PR a fix that solves the problem for your particular PDF.
